udata/frontend/markdown.py

```diff
@@ -1,18 +1,52 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from urlparse import urlparse
+
 import bleach
 import CommonMark
 from html5lib.sanitizer import HTMLSanitizer
 from flask import current_app, Markup
 from werkzeug.local import LocalProxy
 from jinja2.filters import do_truncate, do_striptags
 
+from udata.i18n import _
+
 md = LocalProxy(lambda: current_app.extensions['markdown'])
 
 EXCERPT_TOKEN = '<!--- --- -->'
 
 
+def avoid_mailto_callback(attrs, new=False):
+    """Remove completely the link containing a `mailto`."""
+    if attrs['href'].startswith('mailto:'):
+        return None
```

Comment:
Mostly because I can't use the `safe` filter after content posted by user and I *guess* that 90% of the time the user doesn't want a link at that place. But I may be wrong. Actually I'm not even sure if we have that case today…